src/main/java/server/ServerHandler.java

```diff
@@ -2,41 +2,51 @@
 
 import client.User;
 import client.Validator;
-import codec.Session;
+import room.Room;
+import server.commands.*;
```

Comment:
Try not to use wildcards for imports. It makes imports shorter but it is become harder to understand what is the class you use when somebody reviewers your codec in  github for example